shopinvader_locomotive/models/shopinvader_variant.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import api, exceptions, fields, models, _
+
+
+class ShopinvaderVariant(models.Model):
+    _inherit = 'shopinvader.variant'
+
+    @api.multi
+    def _unbind(self):
+        """
+        Inherit to call the delete on locomotive if the recordset is un-binded.
```

Comment:
@acsonefho This comment should be wrong since no variant are exported to locomotive...